packages/core/admin/admin/src/components/LeftMenu.tsx

```diff
@@ -142,7 +142,12 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }: LeftMenuProps) =
               label={formatMessage({ id: 'global.home', defaultMessage: 'Home' })}
             >
               <NewNavLink.Icon>
-                <Icon as={House} color="neutral500" />
+                <Icon
+                  width={`${16 / 16}rem`}
```

Comment:
no need of 16/16, you can say `1rem` but this isn't 2rem?

Reply:
I am following other icons size, I am gonna change the dimensions of the NavLink in the coming ticket CS-758 so please at the moment stick just with the behaviour of the icon